smithy-aws-protocol-tests/model/awsJson1_0WithQueryCompatible/main.smithy

```diff
@@ -0,0 +1,18 @@
+$version: "2.0"
+
+namespace aws.protocoltests.json10
+
+use aws.api#service
+use aws.protocols#awsJson1_0
+use aws.protocols#awsQueryCompatible
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+
+
+@service(sdkId: "JSON10 WITH QUERY COMPATIBLE TRAIT")
```

Comment:
nit: this should be un-caps'd and updated to match others - "JSON RPC 10 with Query Compatible Trait"